plugins/GrapesJsBuilderBundle/Config/config.php

```diff
@@ -3,7 +3,7 @@
 declare(strict_types=1);
 
 return [
-    'name'        => 'Builder',
+    'name'        => 'GrapesJS Builder',
     'description' => 'GrapesJS Builder with MJML support for Mautic',
     'version'     => '1.0.0',
```

Comment:
```suggestion
```
Since the composer work we get the version from [Packagist](https://packagist.org/packages/mautic/grapes-js-builder-bundle) - I think therefore that we should not be explicitly stating it in this way

Reply:
Are we sure the plugin bundle do not need version? If we need made process some migration during plugin update? I think yes.

During the `mautic:plugins:reload` Mautic check plugin version and If version was updated then execute some code to run plugin migration etc...

Reply:
I did check that with @nickveenhof  but let's double check!

Reply:
I am talking about this part - https://developer.mautic.org/#plugin-schema
Without plugin versioning, it's not possible do it.